### DISCUSSION TITLE
Plot Heatmap for Positive TF-Regulators

### DISCUSSION THREAD

**Discussion by Sky970415** [GreenleafLab/ArchR#2032] [Feature Requests]
Hi,
In the full tutorial manual, positive TF-regulators can be only shown as lists. In the trajectory section, there is a heatmap that can put motif heatmap along side with gene expression heatmap. It make any sense for positive TF-regulator to have a similar heatmap for manuscripts. 

I modified the function for trajectory heatmap to plot positive TF-regulators. Which file I should put this function into for pull request? Any other protocol I need to follow for the contribution?

Thanks,
Jiakang

**Comment by rcorces** [comment]
pull requests should be made into the `dev` branch. It would be helpful to see the output of the function on the tutorial data or at least some other data set. Thanks!
